manager/Makefile

```diff
@@ -8,17 +8,19 @@ DOCKER_NAME ?= manager
 KUBE_NAMESPACE ?= fybrik-system
 CONTROLLER_NAMESPACE ?= ${KUBE_NAMESPACE}
 
+GIT_COMMIT := "$(shell git rev-list -1 HEAD)"
```

Comment:
@cdoron , please remove quotes around the shell command, otherwise we get double quotes in the build and run rules